app/models/taxon.rb

```diff
@@ -1,5 +1,7 @@
 class Taxon
+  # rubocop:disable Lint/MixedRegexpCaptureTypes
```

Comment:
Could we explain in the commit message why we're ignoring this? I can see at least one redundant capture group (around the "prefix" part).

Reply:
Thanks, good prompt - I looked and we don't need to ignore this, I've changed to a non-capturing group to resolve the offence and replaced the commit.

Reply:
Nice, thanks 👍.